### ISSUE TITLE
Gunbad: Left wing bosses said locked out when didn't have none and no boss spawn

### ISSUE THREAD

**Issue by emailkook** [WarEmu/WarBugs#16575]
We did Gunbad right wing without issues. Only 2 group members had middle lockouts. I haven't done left wing for a week. 3 of the sub-bosses in the left wing gave me a message saying due to 2 or more having lockouts, I passed on loot. However the group members got their loot rolls but I didn't. Almost seems like as it's giving credit down the list to group members, by the time it comes to me it thinks I did the encounter w/ group members being locked out. 

When we got to the last boss by zoning through, he was missing and everyone was confused as there are no lockouts on anyone.

Each boss should've given me a loot roll as the expected behavior having no .lockout or in the Character Sheet Lockout tab. Actual behavior was getting no loot and no quest credit either. Maybe due to new patch it used old data from last week when I did it and used that lockout timer? Guess I try again in 2 days 14 hours when this new lockout is cleared. As you can see from the screenshot, it's a fresh lockout I just got from the left wing but yet zero reward or credit thinking I'm locked.

![Spell_004](https://user-images.githubusercontent.com/50253609/80656355-a2afde80-8a46-11ea-91f9-0e71aff0d04a.jpg)

**Comment by Kaelang** [comment]
Just ran all three wings and worked fine for me.

Please can you confirm if this happens following the next lockout.

**Comment by FranchiseRoR** [comment]
I experienced the same issue. Earlier today I had attempted Gunbad left wing but experienced https://github.com/WarEmu/WarBugs/issues/16567 there. The group had decided to attempt Mid despite that bug. We started Redeye Big Oaf but never killed him due to server lag and disconnects and disbanded the group. Several hours later I'm looking at my lockouts and notice that I have lockouts for bosses we didn't kill in mid including Redeye Big Oaf and several others. Only bosses we had killed were in the left wing.

![Mystery Lockouts](https://user-images.githubusercontent.com/36144491/80664229-33e17e00-8a64-11ea-9bd4-3c2d4245bc17.png)

We had changed leaders when attempting mid, is it possible I was tied to his instance so he returned later and killed those bosses. Or possibly tied to instances of others from my party?

**Comment by emailkook** [comment]
Add what he said to my report too. When I started Gunbad I had NO lockouts
other than Crypts. After the whole left wing fiasco, the group leader said
I have middle wing lock out too. I didn’t think about it until reading his
issues too. Why did it give me Big Oaf and other middle lock out???

Sounds like it’s picking one group member and randomly assigning them to
someone else in the database or kills on left wing also giving middle wing
lockouts too.

Something is really wrong and weird. Kae try doing it as a different group
member in the order. Maybe the 5th or 6th invited one to group? I also did
a rename after the patch if that contributed anything too. L

**Comment by DmitriyMaksimovich** [comment]
The same error, I did not start the game for 6 days, but when I entered the left wing, I got lockouts on all the bosses.

**Comment by emailkook** [comment]
Today we started a middle and left Gunbad raid. After first PQ boss kill
someone on discord said “wtf I haven’t done it in a week and it says I was
locked out”. Rolled loot for only 5 people. Exact same thing happened to me
on the left wing which I started the bug report about.

I’m telling you, it picks one person in the group and gives them a lockout
on EVERY boss on that side even though they’re 100% clear. I DVR’d that
moment and will upload the clip here when I get home.

You can hear his reaction and see his name missing from the rolls.
Hopefully that helps you figure out why one person is always getting a full
lockout. So it’s not just related to left side since today it happened to
center.

Might be related to their position in the group.

**Comment by grunbag** [comment]
someone had the lockout in your group. make sure to type .lockout before starting a dungeon.

**Comment by emailkook** [comment]
The last raid he was locked out right wing. Previous raid when we did center no one had lockout and he claimed didn’t even do a Gunbad for one week. When I originally posted this I didn’t do ANY Gunbad for ONE week. We did left and all PQ were telling me locked out AND he added a Big Oaf lockout from center without even doing it. Another posted claiming same thing getting a Big Oaf lockout from left won’t.  I’ve been a tester for Warhammer live internally and I can tell you these bugs are real seeing 3 groups I’ve joined in 1 1/2 weeks and each time someone was left out and lost their mind. I don’t make bug reports lightly and complain. This is a valid but but now that I’m done with Gunbad be hard to report it for now. Good news... final boss still gives you group quest to the person that was locked out from PQ. I’m thinking when they did a different wing PQ, since drops are the same with boots and gloves, it’s sharing the lockout and carrying it out. I disagree with closing this report but it is what it is and next type will post pre-raid .lockout and post. I did check lockouts before raid. Thank you
